framework/contracts/account/src/modules.rs

```diff
@@ -657,196 +657,148 @@ mod tests {
-            };
-
-            // -1 because manager counts as module as well
-            for i in 0..LIST_SIZE_LIMIT - 1 {
-                let test_module = format!("module_{i}");
-                let test_module_addr = deps.api.addr_make(&test_module);
-                msg = ExecuteMsg::AddModules {
-                    modules: vec![test_module_addr.to_string()],
-                };
-                let res = execute_as_admin(&mut deps, msg.clone());
-                assert_that(&res).is_ok();
-            }
-
-            let res = execute_as_admin(&mut deps, msg);
-            assert_that(&res)
-                .is_err()
-                .is_equal_to(ProxyError::ModuleLimitReached {});
-        }
-    }
-
-    type ProxyTestResult = Result<(), ProxyError>;
-
-    mod remove_module {
-        use abstract_std::proxy::state::State;
-        use cw_controllers::AdminError;
-
-        use super::*;
-
-        #[test]
-        fn only_admin() {
-            let mut deps = mock_dependencies();
-            mock_init(&mut deps);
-
-            let msg = ExecuteMsg::RemoveModule {
-                module: TEST_MODULE.to_string(),
-            };
-            let info = message_info(&deps.api.addr_make("not_admin"), &[]);
-
-            let res = execute(deps.as_mut(), mock_env(), info, msg);
-            assert_that(&res)
-                .is_err()
-                .is_equal_to(ProxyError::Admin(AdminError::NotAdmin {}))
-        }
-
-        #[test]
-        fn remove_module() -> ProxyTestResult {
-            let mut deps = mock_dependencies();
-            mock_init(&mut deps);
-
-            let test_module_addr = deps.api.addr_make(TEST_MODULE);
-            STATE.save(
-                &mut deps.storage,
-                &State {
-                    modules: vec![test_module_addr.clone()],
-                },
-            )?;
-
-            let msg = ExecuteMsg::RemoveModule {
-                module: test_module_addr.to_string(),
-            };
-            let res = execute_as_admin(&mut deps, msg);
-            assert_that(&res).is_ok();
-
-            let actual_modules = load_modules(&deps.storage);
-            assert_that(&actual_modules).is_empty();
-
-            Ok(())
-        }
-
-        #[test]
-        fn fails_removing_non_existing_module() {
-            let mut deps = mock_dependencies();
-            mock_init(&mut deps);
-
-            let test_module_addr = deps.api.addr_make(TEST_MODULE);
-            let msg = ExecuteMsg::RemoveModule {
-                module: test_module_addr.to_string(),
-            };
-
-            let res = execute_as_admin(&mut deps, msg);
-            assert_that(&res)
-                .is_err()
-                .is_equal_to(ProxyError::NotWhitelisted(test_module_addr.to_string()));
-        }
-    }
+    // TODO: move these tests to integration tests
+
+    // mod add_module {
+    //     use super::*;
+
+    //     use cw_controllers::AdminError;
+
```

Comment:
This TODO will be done in different PR?

Reply:
Yes, let's 

I think this was already commented out before the refactor